test/crawler_web/features/home_page/view_keyword_list_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule CrawlerWeb.Features.HomePage.ViewKeywordListTest do
+  use CrawlerWeb.FeatureCase, async: false
+
+  feature "view list of uploaded keywords table for a user", %{session: session} do
+    user = insert(:user)
+    insert(:keyword, user_id: user.id, name: "buy phone")
+    insert(:keyword, user_id: user.id, name: "buy tv")
+
+    session
+    |> log_in_user(user)
+    |> visit(Routes.home_path(CrawlerWeb.Endpoint, :index))
+    |> assert_has(Query.text(gettext("buy phone")))
+    |> assert_has(Query.text(gettext("buy tv")))
```

Comment:
Do we need to use the gettext for these texts? because that is the text from our database

```suggestion
    |> assert_has(Query.text("buy phone"))
    |> assert_has(Query.text("buy tv"))
```